source/includes/concept/_metadata.md

```diff
@@ -0,0 +1,20 @@
+## Metadata
+
+Understanding the context of data, empowering users to find and make informed decisions about it's suitability, is another important goal of the Resource Watch API (RW API). For example a user maybe looking to answer questions like *"Does this dataset provide information about tree cover in my region?"*, *"What are the physical units?"*, *"How was this measured?"*, and *"How should I provide proper attribution to the data provider?"*.
+
+These questions imply being able to access summary data about the data, so called [Metadata](https://guides.ucf.edu/metadata/intrometadata). By definition metadata is always associated with another entity, such as a dataset, layer or widget, and maybe further classified into different types such as descriptive, structural, and/or administrative. Many different general and discipline-specific metadata standards and schemas are [available](http://rd-alliance.github.io/metadata-directory/standards/) each with their pros and cons. However, in general, the type of information desired about data is:
+
+- **Title/Name** – Name given to the resource.
+- **Description** – A description of the resource, its spatial, temporal or subject coverage, and it’s methodology ad lineage.
+- **Format** – File format, physical medium, dimensions of the resource, or hardware and software needed to access the data.
+- **Metadata schema** – Description of the metadata to be provided along with the generated data and a discussion of the metadata standards used, including the version of the schema and where the schema can be found.
+- **Identifier** – A unique identification assigned to the resource.
+- **Rights Holder** – The entities or persons who hold the rights to the data.
+- **Rights** – Information about the rights held in and over the resource.
+- **Contact Information** – Identity of, and means to communicate with persons or entities associated with the data.
```

Comment:
I really like this introduction, I think it goes to the point of explaining what metadata is and how it is useful. However, I get a bit lost by the end of the section - I think we are missing the connection between "metadata" as a general concept and "metadata" in the context of the RW API.

I'd add a sentence here stating that, in the RW API, we have tried to follow the main principles that research suggests about metadata while keeping the structure as flexible as possible. This would also be a good segway for the next paragraph, where you explain some of the requirements of metadata for the RW API.

Reply:
done